admin_manual/configuration_ldap/examples/multi-server.slapd.conf

```diff
@@ -1,49 +1,84 @@
+//Global Directives:
+//Schema and objectClass definitions
 include         /etc/ldap/schema/core.schema
 include         /etc/ldap/schema/cosine.schema
 include         /etc/ldap/schema/nis.schema
 include         /etc/ldap/schema/inetorgperson.schema
 
+//Where the pid file is put. The init.d script
+//will not stop the server if you change this.
 pidfile         /var/run/slapd/slapd.pid
+
+//List of arguments that were passed to the server
 argsfile        /var/run/slapd/slapd.args
 
+//Read slapd.conf(5) for possible values
 loglevel        any
 
+//Where the dynamically loaded modules are stored
 modulepath      /usr/lib/ldap
 moduleload      rwm
 moduleload      pcache.la
 moduleload      back_ldap.la
 moduleload      back_meta.la
 moduleload      back_hdb.la
 
+//The maximum number of entries that is returned for a search operation
 sizelimit       500
+
+//The tool-threads parameter sets the actual amount of cpu's that is used
```

Comment:
I think the original comment was wrong, at least according to [the slapd.conf man pages](https://linux.die.net/man/5/slapd.conf). It uses the text: 

> Specify the maximum number of threads to use in tool mode. This should not be greater than the number of CPUs in the system. The default is 1.